src/styles/card.scss

```diff
@@ -0,0 +1,95 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 198px;
+  height: 408px;
```

Comment:
do not hardcode height - it should depend on the content - if you set the correct margins and text/images sizes inside it will match the design